server/routes/index.ts

```diff
@@ -2,15 +2,27 @@ import { type RequestHandler, Router } from 'express'
 
 import asyncMiddleware from '../middleware/asyncMiddleware'
 import type { Services } from '../services'
+import InputsController from '../controllers/inputsController'
+import formatDate from '../utils/dateHelpers'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export default function routes(service: Services): Router {
   const router = Router()
   const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
-
+  const post = (path: string | string[], handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
   get('/', (req, res, next) => {
     res.render('pages/index')
   })
 
+  get('/inputs', InputsController.getInputs)
+
+  post('/inputs', InputsController.saveInputs)
+
+  get('/services', (req, res, next) => {
+    res.render('pages/inputs', {
+      today: formatDate(new Date().toISOString(), 'short'),
```

Comment:
Not sure if it's a concern as we only expect users access our service during day time, but should we include toISOString("en-UK", {timeZone: "Europe/London"}) to make sure the date is always London time?

Reply:
This is always based on server time since this runs in kubernetes rather than in the client's browser, so I think it's fine without.